src/cookbook/navigation/passing-data.md

```diff
@@ -6,7 +6,7 @@ prev:
   path: /cookbook/navigation/returning-data
 next:
   title: Delete data on the internet
-  path: /cookbook/networking/delete-data
+  path: /cookbook/navigation/set-up-universal-links
```

Comment:
This path doesn't seem to match the title

Reply:
nice catch!